applications.md

```diff
@@ -0,0 +1,77 @@
+# DLS Applications
+
+[Architecture diagram](https://docs.google.com/drawings/d/1qqHoceL4nahv8wmhK_QltL8f1StdBJ5GYFpIa6JQ3PA/edit)
+
+## in Production
```

Comment:
Add the IIIF image server? https://github.com/pulibrary/serverless-iiif

No Liaison or PO though, and it might not make sense to have them. Maybe tooling? I dunno. At least having it on here with those properties filled in as TBD would be good.